hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/ScheduleFeature.java

```diff
@@ -60,26 +63,74 @@ public class ScheduleFeature {
     private NetworkTransactionResponse networkTransactionResponse;
     private ScheduleId scheduleId;
     private TransactionId scheduledTransactionId;
+    private Long plusSecondsToExpire;
+    private String scheduleTxConsensusTimestamp;
+
+    @Given(
+            "I successfully schedule a HBAR transfer from treasury to {account} {string} expiration time and wait for expiry {string} - plus {int} seconds")
```

Comment:
It might be clearer where expiration time string can be `null` or a duration offset like `2s`. Can use `DurationStyle.detectAndParse(str)` for parsing simple duration strings.

```suggestion
            "I successfully schedule a HBAR transfer from treasury to {account} with  expiration time {string} and wait for expiry {string}")
```

Reply:
Changed it that way